src/mlpack/methods/ann/layer/reinforce_normal_impl.hpp

```diff
@@ -34,8 +34,7 @@ void ReinforceNormal<InputDataType, OutputDataType>::Forward(
   if (!deterministic)
   {
     // Multiply by standard deviations and re-center the means to the mean.
-    output = arma::randn<arma::Mat<eT> >(input.n_rows, input.n_cols) *
-        stdev + input;
+    output = output.randn(input.n_rows, input.n_cols) * stdev + input;
```

Comment:
I didn't know `randn()` returned the matrix so that it could be used like this.  Nice!

Reply:
thank you!

Reply:
Same here, actually I had to test it out.

Reply:
Yes, I also tested it and it seems to return a matrix. did it happen to behave the same ways?

Reply:
I believe it is :))

Reply:
Yep, looks great, thank you!